README.md

```diff
@@ -2,7 +2,7 @@
 
 A list of developer's personal websites that are interesting in their own way: design, originality, craziness or simply effectiveness.
 
-
+- [Adam Greenough](https://adgr.dev/)
```

Comment:
Please make sure to add your website to the **end** of the list as they have been added chronologically ! Thank you.

Reply:
Opened new PR for end!